docs/api/_src/conf.py

```diff
@@ -20,9 +20,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "nf-core tools API"
-copyright = "2019, Phil Ewels, Sven Fillinger"
-author = "Phil Ewels, Sven Fillinger"
+project = "nf-core/tools"
+copyright = "2019, nf-core community"
```

Comment:
I am never sure about the copyright notices. Since we update the text shouldn't we then also update the copyright to the current year?

Reply:
Yeah maybe. And I'm not sure what is even valid or if it's useful for anything.